### DISCUSSION TITLE
Unraid Docker Image Not Available

### DISCUSSION THREAD

**Discussion by wraith1385** [Taxel/PlexTraktSync#1395] [Support] [answered]
The Unraid Docker Image is now showing as not available when you check for updates.  The container still runs fine in it's current state, but stopped receiving updates a week ago.  When you look at it in the Unraid Apps tab, the Last Update is unknown.

**Comment by tyranian** [comment]
I think this is an issue with the latest tag (ghcr.io/taxel/plextraktsync:latest) as when you use a specific number it works i.e ghcr.io/taxel/plextraktsync:0.25.15 it will update, but this means updating the tag everytime you want to do an update.

**Reply by momoz** [reply, marked as answer]
I use the latest tag and it will have an issue with the update button.  Choose advanced and force update and it will update.   Also as of the latest mine now says Up to date instead of unknown.

**Reply by tyranian** [reply]
thanks that's a better way to do it!
